modules/lfs/server.go

```diff
@@ -233,6 +233,12 @@ func PostHandler(ctx *context.Context) {
 		return
 	}
 
+	if setting.LFS.MaxFileSize > 0 && rv.Size > setting.LFS.MaxFileSize {
+		log.Info("Denying LFS upload of size %d (>%d) to %s/%s", rv.Size, setting.LFS.MaxFileSize, rv.User, rv.Repo)
+		writeStatus(ctx, 403)
```

Comment:
finally been able to check the status that Github returns I think this should be: 413

```suggestion
		writeStatus(ctx, 413)
```